aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -76,6 +81,17 @@ func resourceAwsApiGatewayRestApiCreate(d *schema.ResourceData, meta interface{}
 
 	d.SetId(*gateway.Id)
 
+	if body, ok := d.GetOk("body"); ok {
+		_, err := conn.PutRestApi(&apigateway.PutRestApiInput{
```

Comment:
Could you log in the logs that we are setting the specification?
Just for verbosity & debugging :D

Same for the update would be highly appreciated!